### PR TITLE
Up 5 profile view

### PR DESCRIPTION
**Pull request by AlexDjangoX**
User can type id into URL:

(http://localhost:3000/profile/6) 

and see user profile on UI

### CODE REVIEW

**Review by Auenc — APPROVED** [review]
LGTM

File: src/App.js [hunk 1]
```diff
@@ -24,6 +25,8 @@ function App() {
           <Route element={<AuthenticateUser />}>
             <Route path='/posts' element={<PostsPage />} />
           </Route>
+          <Route path='/edit-profile' element={<EditProfile />} />
```
Comment (Auenc) [inline src/App.js:28]: These routes should be nested inside the `<Route element={<AuthenticateUser />}>`

**Review by Auenc — CHANGES_REQUESTED** [review]
Sorry, I missed these. 

The routes you are adding should be inside the authenticated route as you should only be able to access them when a user is signed in.

**Review by Auenc — APPROVED** [review]
LGTM

**Review by sigristarisa — APPROVED** [review]

**Review by dogezen — APPROVED** [review]

**Pull request merged**